packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -2470,15 +2471,16 @@ export class ModularEditBuilder extends EditBuilder<ModularChangeset> {
 	public buildTrees(
 		firstId: ChangesetLocalId,
 		content: ITreeCursorSynchronous,
+		idCompressor?: IIdCompressor,
 	): GlobalEditDescription {
 		if (content.mode === CursorLocationType.Fields && content.getFieldLength() === 0) {
 			return { type: "global" };
 		}
 		const builds: ChangeAtomIdBTree<TreeChunk> = newTupleBTree();
 		const chunk =
 			content.mode === CursorLocationType.Fields
-				? chunkFieldSingle(content, defaultChunkPolicy)
-				: chunkTree(content, defaultChunkPolicy);
+				? chunkFieldSingle(content, { policy: defaultChunkPolicy })
```

Comment:
is there a reason this one is not given the idCompressor?

If so, a comment on it would be helpful.

If not, saving the chunk compressor to a local variable  and using the same one for both to deduplicate its declaration would be nice.

Reply:
Saved the chunk into a local variable for simplification